assets/sass/components/adsense/_googlesitekit-adsense-ad-blocking-recovery.scss

```diff
@@ -82,11 +82,14 @@
 
 	.googlesitekit-ad-blocking-recovery__footer-cancel {
 		border-top: 1px solid $c-mercury;
-		padding: 17px;
+		padding-top: 14px;
 
 		@media (min-width: $bp-desktop) {
 			border: none;
-			padding: 0;
+		}
+
```

Comment:
![image](https://github.com/google/site-kit-wp/assets/20284937/0e42374e-9a0f-4ff6-9f27-a333bf5cb0f3)

It looks like the `Cancel` action is slightly misaligned vertically.

![image](https://github.com/google/site-kit-wp/assets/20284937/ce9767ec-a218-4437-8648-244a1336e015)

Also, in smaller viewports, the border doesn't cover the entire width of the container.

Could we remove the border and padding entirely from `.googlesitekit-ad-blocking-recovery__footer-cancel` and let `.googlesitekit-ad-blocking-recovery__footer` handle that instead?

Reply:
Done, way better, thanks 🙏🏻